<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditDynamicPlayList</class>
 <widget class="QDialog" name="DialogEditDynamicPlayList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dynamic playlist</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name of the list:</string>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName">
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="editLimit">
       <property name="text">
        <string>Limit to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="editNumItems">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="editItemsType">
       <item>
        <property name="text">
         <string>songs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gio</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>selected by</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="editSelected">
       <item>
        <property name="text">
         <string>title</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>artist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>album</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>genre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>highest rating</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lowest rating</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>most recently played</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>least recently played</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>most often played</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>least often played</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="editOnlyChecked">
     <property name="text">
      <string>Match only checked items.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="editUpdate">
     <property name="text">
      <string>Live updating.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
